overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -127768,7 +127774,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Finally! Its light shines through!",
-          "desc:8": "Craft a Beacon! You\u0027ll eventually need 4 of them to upgrade your Blood Magic Altar to Tier 5! No need to hurry though. Kill some Withers and explore more of Atum in your free time to get more of its crafting ingredients!"
+          "desc:8": "Craft a Beacon! You\u0027ll eventually need 4 of them to upgrade your Blood Magic Altar to Tier 5! No need to hurry though. Explore Apalachia dungeons for radiant beacons and kill withers for nether stars!"
```

Comment:
since this is an alternative to exploring Atum, that should be referenced
and as before, capitalizing the names of items - Radiant Beacons, Withers, Nether Stars